apps/api/src/admin/participant_manager.py

```diff
@@ -52,24 +52,15 @@ async def get_non_hackers() -> list[Participant]:
     records: list[dict[str, Any]] = await mongodb_handler.retrieve(
         Collection.USERS,
         {
-            "$and": [
-                {"role": {"$exists": True}},
-                {
-                    "role": {
-                        "$not": {
-                            "$regex": "applicant|"
-                            + "|".join(
-                                (
-                                    Role.DIRECTOR,
-                                    Role.REVIEWER,
-                                    Role.CHECKIN_LEAD,
-                                    Role.ORGANIZER,
-                                )
-                            )
-                        }
-                    }
-                },
-            ]
+            "role": {
+                "$in": [
+                    Role.JUDGE,
+                    Role.SPONSOR,
+                    Role.MENTOR,
+                    Role.VOLUNTEER,
+                    Role.WORKSHOP_LEAD,
+                ]
```

Comment:
Sorry for the merge conflicts but could you rebase with latest `main` and then refactor the new `allowed_roles` from #362 into a shared global?